internal/state/rollback.go

```diff
@@ -41,6 +41,10 @@ func Rollback(bs BlockStore, ss Store) (int64, []byte, error) {
 	if rollbackBlock == nil {
 		return -1, nil, fmt.Errorf("block at height %d not found", rollbackHeight)
 	}
+	lastBlock := bs.LoadBlockMeta(invalidState.LastBlockHeight)
+	if lastBlock == nil {
+		return -1, nil, fmt.Errorf("block at height %d not found", invalidState.LastBlockHeight)
+	}
```

Comment:
Can we call this `latestBlock` and add a comment that the rollbackBlock is 1 before the latest block.

Reply:
done